### PR TITLE
Corkboard 1

### PR DESCRIPTION
**Pull request by chrislloyd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,63 +34,106 @@ If you're just getting starting, it's recommended that you copy the [Webpack exa
...
+group('Basics', () => {
+  ns('Colors');
+  card('Primary Colors', ...); // Goes in the "Basics" -> "Getting Started" group
+});
+```
+
+
+## Standard Theme
+
+The Standard theme is bundled with Corkboard and is meant to facilitate rapid prototyping. It's what you've seen in all the screenshots above. It's meant to be more opinionated than the base Corkboard package — it's helpers will let you move quickly but it mightn't be as flexible as you It comes with helpers built into to help display common types of data.
```
Comment (christianvuerings) [inline README.md]: `it's helpers` - `its helpers` - also the last sentence probably needs some work: `it mightn't be as flexible as you It comes with helper`

File: example/index.js [hunk 2]
```diff
@@ -16,8 +17,133 @@ function Button(props) {
...
+}
+
+card(`
+  test
+`);
+
+group('Getting Started', () => {
+  title('Installation');
+  doc`
+Do this yo.
```
Comment (christianvuerings) [inline example/index.js]: 😎

File: example/webpack.config.js [hunk 3]
```diff
@@ -16,16 +17,29 @@ module.exports = {
...
           ],
         },
       },
+      {
+        test: /\.css$/,
+        // include: [
+        //   path.join(path.dirname(require.resolve('highlight.js')), 'styles'),
+        // ],
+        loaders: [
+          require.resolve('style-loader'),
```
Comment (christianvuerings) [inline example/webpack.config.js]: @chrislloyd does this mean we'll have FOUC? Or do we add these `style` tags on build time?

File: src/standard/Theme.jsx [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+const NavGroup = ({ card }: { card: Card }) => {
+  if (card.type === 'card') { return <div />; }
+
+  const { label, children } = card;
+  const groups = children.filter(c => c.type === 'group');
+  return (
+    <div className={groupStyle}>
+      {label ? <div className={groupLabelStyle}>{label}</div> : null}
+      {groups.map((g, i) => {
+        if (g.children.some(c => c.type === 'card')) {
```
Comment (christianvuerings) [inline src/standard/Theme.jsx]: @chrislloyd do we need this `if`?

File: README.md (hunk 1 above)
Comment (chrislloyd) [inline README.md]: Yeah this is a little out of date (and unpolished) — updates coming soon.

File: example/webpack.config.js (hunk 3 above)
Comment (chrislloyd) [inline example/webpack.config.js]: This isn't actually needed. Will clean up the example.

File: src/standard/Theme.jsx (hunk 4 above)
Comment (chrislloyd) [inline src/standard/Theme.jsx]: Yep, it's just checking if any of it's children are cards, then it shows a link instead of the group label.

**Pull request merged**